packages/next-auth/src/nextAuthPlugin.ts

```diff
@@ -3,39 +3,71 @@ import {
   PartialStlContext,
   MakeStainlessPlugin,
   Params,
+  StlContext,
 } from "stainless";
 import { IncomingMessage, ServerResponse } from "http";
 import { NextServerContext } from "@stl-api/next";
 import { AuthOptions, Session, getServerSession } from "next-auth";
 
+/// An authenticated session, with user data available.
+export type UserSession = Session & {
+  user: {
+    name?: string | null
+    email?: string | null
+    image?: string | null
+  }
```

Comment:
Should this be more like this?
```suggestion
  user: {
    name: string | null
    email: string | null
    image: string | null
  }
```
or even without the `?` after `user`?

I guess I'm confused why they keys would be undefined here…

Reply:
That's the way `user` is defined in `Session`. I didn't want to bake in assumptions on how `user` looks like; unfortunately, I have to inline the user type from `Session` because it isn't separately defined.

Reply:
Just to be clear, the util I suggested would set `user` to not have a `?`, but wouldn't set the child properties to be certainly present.

Reply:
Eh, what you have is fine